kotest-property/src/commonMain/kotlin/io/kotest/property/arbitrary/arbs.kt

```diff
@@ -9,17 +9,29 @@ import io.kotest.property.Shrinker
 import io.kotest.property.map
 import io.kotest.property.sampleOf
 
+/**
+ * Returns a sequence from values generated from this arb.
+ * Edgecases will be ignored.
+ */
+fun <A> Arb<A>.randomSequence(rs: RandomSource = RandomSource.Default): Sequence<A> {
```

Comment:
This exists to do the logic of which method to use - value or values. Given that its just a workaround might it be clearer to have this in the arb class itself so it's easy to see that this should be used and not values when adding new arbs.

Reply:
Awesome I'll make that change